### PR TITLE
Fix various spacing and capitalization issues

### PR DESCRIPTION
**Pull request by Calinou**

### CODE REVIEW

File: tutorials/shading/shading_reference/shading_language.rst [hunk 1]
```diff
@@ -586,7 +586,7 @@ Full list of hints below:
 +================+=============================+=====================================+
 | **vec4**       | hint_color                  | Used as color                       |
 +----------------+-----------------------------+-------------------------------------+
-| **int, float** | hint_range(min,max [,step]) | Used as range (with min/max/step)   |
+| **int, float** | hint_range(min, max[, step]) | Used as range (with min/max/step)   |
```
Comment (mhilbrunner) [inline tutorials/shading/shading_reference/shading_language.rst:589]: I think this one breaks the table layout and the table needs to be adjusted? :)

**Comment by mhilbrunner** [comment]
Only spotted the table thing, everything else looks good to go I think :)

File: tutorials/shading/shading_reference/shading_language.rst (hunk 1 above)
Comment (Calinou) [reply tutorials/shading/shading_reference/shading_language.rst:589]: Fixed.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**